iac-extensions/docker/src/main/java/org/sonar/iac/docker/tree/api/AddTree.java

```diff
@@ -19,10 +19,5 @@
  */
 package org.sonar.iac.docker.tree.api;
 
-import java.util.List;
-
-public interface AddTree extends InstructionTree {
-  List<ParamTree> options();
-  List<SyntaxToken> srcs();
-  SyntaxToken dest();
+public interface AddTree extends CommandSrcsDestOptionInstructionTree {
```

Comment:
:exploding_head: I had to read this class name 10 times :smile: 
I would choose something like `TransferInstructionTree` because both instruction transfer elements from the local to the Docker file system.

Reply:
I'm especially bad at finding names ^^' After spending several minutes I surrender and fall back on some poor choice I find.
I will take the `TransferInstructionTree` 👍